misc/python/materialize/mzcompose/services/balancerd.py

```diff
@@ -31,11 +34,17 @@ def __init__(
                 "--https-resolver-template='materialized:6876'",
             ]
 
+        depends_graph: dict[str, ServiceDependency] = {
+            s: {"condition": "service_started"} for s in depends_on
+        }
+        config: ServiceConfig = {
+            "mzbuild": mzbuild,
+            "command": command,
+            "ports": [6875, 6876, 6877, 6878],
```

Comment:
MZ_BALANCER_SQL/HTTP_LISTEN_ADDR ports are 6880/6881 though. Do we not want to expose them?
Edit: seen later, those are exposed in environmentd.